### PR TITLE
Avoid javascript errors and use invariant culture to avoid formating error...

### PR DESCRIPTION
**Pull request by Johnny-Bee**
visualobjects.js:  2 patchs to avoid a lot of javascript errors !!
Color.cs and Coordinate.cs: use invariant culture to avoid error due to culture formating ie french decimal separator :-)
Index.html: Add <!DOCTYPE html> to avoid warning from the developer's console
VisualObjects.WebService.csproj: change CopyToOutpuDirectory Always by security.
Works well ! http://i.imgur.com/eun8UND.png

### CODE REVIEW

File: Actors/VisualObjects/VisualObjects.WebService/wwwroot/Index.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿<html>
+﻿<!DOCTYPE html>
```
Comment (vturecek) [inline Actors/VisualObjects/VisualObjects.WebService/wwwroot/Index.html]: Setting the doctype actually messes with the rendering of the box, making it super small. You can tell from your screenshot that you had to zoom way in to see the display which shouldn't be the case; it should take up the whole browser window.

Comment (vturecek) [inline Actors/VisualObjects/VisualObjects.WebService/wwwroot/Index.html]: Other changes look good though, if you revert this index.html change, I'd like to merge this!

**Comment by vturecek** [comment]
Thanks for the contribution!

**Comment by masnider** [comment]
+1, Thanks @Johnny-Bee

**Pull request merged**